packages/@okta/i18n/src/properties/login_sv.properties

```diff
@@ -900,6 +901,8 @@ oie.authenticator.enroll.error.fail = Det g\u00E5r inte att registrera autentise
 oie.authenticator.verify.error.fail = Det g\u00E5r inte att verifiera autentiseraren. F\u00F6rs\u00F6k igen.
 oie.configuration.error = N\u00E5gonting gick fel. Eventuell felkonfigurering detekterad. Kontakta support.
 oie.auth.logo.aria.label = Autentiseringslogotyp
+oie.challenge.answer.showAnswer = Visa svaret
+oie.challenge.answer.hideAnswer = Hide answer
```

Comment:
Translations on this file are in English, which is wrong. Let me figure out what happened

Reply:
We saw this for other *_sv files as well this month